packages/api-types/types/api.d.ts

```diff
@@ -4451,6 +4485,7 @@ export interface components {
         | 'rls_references_user_metadata'
         | 'materialized_view_in_api'
         | 'foreign_table_in_api'
+        | 'unsupported_reg_types'
```

Comment:
the only relevant one for this PR, but assuming this file hasn't been re-gen'd in a while